xbmc/guilib/GUITexture.h

```diff
@@ -56,6 +56,7 @@ class CTextureInfo
   CTextureInfo& operator=(const CTextureInfo& right) = default;
   bool       useLarge;
   CRect      border;          // scaled  - unneeded if we get rid of scale on load
+  bool       infill = true;   // if false, the main body of a texture is not drawn. useful for borders with no inner filling
```

Comment:
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#114-default-member-initialization

Reply:
I'll try to remember it :).